test/test_tensorexpr.py

```diff
@@ -1056,5 +1057,32 @@ def test(x, y, z):
             np.testing.assert_allclose(ref.cpu().numpy(), res.cpu().numpy())
             assert cuda.elapsed_value() == 1
 
+    @unittest.skipIf(not torch.cuda.is_available(), "requires CUDA")
```

Comment:
It looks like you skip the test if CUDA is not available, but then only test on the CPU device?

Reply:
Thanks for catching this, let me fix it. I was playing with it and left in this state